packages/desktop/views/dashboard/wallet/views/WalletView.svelte

```diff
@@ -0,0 +1,39 @@
+<script lang="ts">
+    import { AssetList, Overflow, Pane, ReceiveAddressButton } from '@ui'
+    import { AccountSummary, AccountActivity, SendButton } from '@components'
+    import { selectedWalletAssets } from '@core/wallet'
+    import { selectedWallet } from '@core/wallet/stores'
+    import features from '@features/features'
+</script>
+
+{#if $selectedWallet}
+    {#key $selectedWallet?.index}
```

Comment:
this if and key could probably be moved to `packages/desktop/views/dashboard/wallet/Wallet.svelte` so both views use it, they will likely both need this condition